### PR TITLE
pizza process

### PR DESCRIPTION
**Pull request by un-install**

### CODE REVIEW

File: src/main/java/lesson20/AdapterExample/Adapter.java [hunk 1]
```diff
@@ -0,0 +1,29 @@
+package lesson20.AdapterExample;
+import lesson20.MamaMia.Peperony;
+import lesson20.Pizza;
+
+public class Adapter extends Pizza{
```
Comment (Taniann) [inline src/main/java/lesson20/AdapterExample/Adapter.java]: Better name from which class to which class you've adapted

File: src/main/java/lesson20/AdapterExample/Adapter.java [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+    SquerePie pie;
```
Comment (Taniann) [inline src/main/java/lesson20/AdapterExample/Adapter.java]: Add access modificator

File: src/main/java/lesson20/Demo.java [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+import lesson20.AdapterExample.SquerePie;
+import lesson20.Factory.AbstractFactory;
+import lesson20.Factory.FactoryProducer;
+import lesson20.ProxyPizza.ProxyPeperony;
+import lesson20.Wrapper.Delivery;
+import lesson20.Wrapper.HotDelivery;
+
+public class Demo {
+    public static void main(String[] args) {
+        Delivery delivery = new Delivery();
```
Comment (SergeyPasko) [inline src/main/java/lesson20/Demo.java]: You use concretic factory|class. It is hardcode. Read https://www.baeldung.com/java-abstract-factory-pattern

Comment (SergeyPasko) [reply src/main/java/lesson20/Demo.java]: Ok, I saw below =)

File: src/main/java/lesson20/AdapterExample/RoundBox.java [hunk 4]
```diff
@@ -0,0 +1,15 @@
+package lesson20.AdapterExample;
+
+import static java.lang.Math.pow;
```
Comment (SergeyPasko) [inline src/main/java/lesson20/AdapterExample/RoundBox.java]: It is not a adapter

Comment (un-install) [reply src/main/java/lesson20/AdapterExample/RoundBox.java]: it is not

File: src/main/java/lesson20/AdapterExample/SquerePieToPizza.java [hunk 5]
```diff
@@ -2,17 +2,24 @@
 import lesson20.MamaMia.Peperony;
 import lesson20.Pizza;
 
+import static java.lang.Math.pow;
+import static java.lang.Math.sqrt;
+
 public class SquerePieToPizza extends Pizza{
     private SquerePie pie;
```
Comment (un-install) [inline src/main/java/lesson20/AdapterExample/SquerePieToPizza.java]: this is adapter

Comment (un-install) [reply src/main/java/lesson20/AdapterExample/SquerePieToPizza.java]: Pizza.getRadius() - target method
pie.getSide() - adapted method

**Pull request merged**